bin/cli.js

```diff
@@ -57,19 +59,103 @@ async function initKnex(env, opts) {
   return knex(resolvedConfig);
 }
 
-function invoke(env) {
-  env.modulePath = env.modulePath || env.knexpath || process.env.KNEX_PATH;
+function findUpModulePath(cwd, packageName) {
```

Comment:
Could you please move all the new util code into separate file(s)?

Reply:
bin/utils/cli-config-utils.js?

Reply:
Not sure if all of new functions are config-related, but yeah, you can introduce new files there as well.

Reply:
modulePath is probably config related